cmds/core/backoff/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2012-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Run a command, repeatedly, until it succeeds or we are out of time
+//
+// Synopsis:
+//	backoff [-t duration-string] command [args...]
+//
+// Description:
+//	backoff will run the command until it succeeds or a timeout has passed.
+//	The default timeout is 30s.
+//	If no args are given, it will just return.
+//
+// Example:
+//	$ backoff echo hi
+//	hi
+//	$
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+var timeout = flag.String("t", "", "Timeout for command")
+
+func runit(timeout string, c string, a ...string) error {
```

Comment:
timeout parsing should happen elsewhere

`func runit(timeout time.Duration, name string, args ...string) error`

Reply:
done